vxlanBridge.go

```diff
@@ -130,6 +130,26 @@ func (self *Vxlan) SwitchConnected(sw *ofctrl.OFSwitch) {
 	self.initFgraph()
 
 	log.Infof("Switch connected(vxlan)")
+
+	// If vlanDb is populated and switch is connected, this implies
+	// we reconnected to a switch again. In this case, we have to
+	// update the current vlan entry to use a new switch object
+	if len(self.vlanDb) != 0 {
+		for vlanId, vlan := range self.vlanDb {
+			log.Debugf(" Updating vlan %s switch object", vlanId)
+			var err error
```

Comment:
Since none of the variables are used outside of the if statements immediately following them, I would get rid of this error variable and have the rest like this (also added error logging):

```go
if vlan.localFlood, err := self.ofSwitch.NewFlood(); err != nil {
	log.Errorf("Unable to assign new switch to vlan %s local flood: %v", vlanId, err)
	return
}

if vlan.allFlood, err := self.ofSwitch.NewFlood(); err != nil {
	log.Errorf("Unable to assign new switch to vlan %s all flood: %v", vlanId, err)
	return
}
```

Also, do we want to `break` instead of `return`?  Do we want to exit the loop at all or try to continue processing the rest of the entries?  `continue`?

Reply:
same question

Reply:
Never mind about the inline assignment thing, you can't do that when assigning to struct members lol

Reply:
Actually I was think to use os.Exit().